crates/astria-sequencer/src/app/execution_state.rs

```diff
@@ -36,14 +36,14 @@ pub(crate) enum ProposalFingerprintData {
 }
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub(crate) struct ProposalFingerprint(ProposalFingerprintData);
+pub(crate) struct ExecutionState(ExecutionFingerprintData);
```

Comment:
add a comment explaining what this type is for (we do pre-execution, etc) as `ExecutionState` is sort of vague/overloaded